src/prep.py

```diff
@@ -448,7 +448,7 @@ async def get_disc(self, meta):
         bd_summary = None  # noqa: F841
         discs = []
         parse = DiscParse()
-        for path, directories, files in os. walk(meta['path']):
+        for path, directories, files in sorted(os.walk(meta['path'])):
```

Comment:
What is this trying to fix?

Reply:
Figured. Never seen the issue myself, but I don't do an awful amount of multidisc either. Cheers.